services/frontend-service/src/ui/Releases.tsx

```diff
@@ -314,9 +316,25 @@ export const sortEnvironmentsByUpstream = (envs: Environment[], distance: EnvSor
 export const Releases: React.FC<any> = (props: { data: GetOverviewResponse }) => {
     const { data } = props;
     const classes = useStyles(data.environments);
+    const [searchParams] = useSearchParams();
+    let apps = Object.values(data.applications);
 
-    const keys = Object.keys(data.applications);
-    keys.sort();
+    if (searchParams.has('app')) {
+        apps = apps.filter((k) => k.name.includes(searchParams.get('app')!));
```

Comment:
Wouldn't it be better to have an actual route to the app? Something like "/applications/<app>" maybe?

Reply:
This is just a filter, since we currently treat / as the list of all applications. So it doesn't make sense to add it as a route. 

When we have details page for applications (v.1.0) it might make sense